src/CoreOperators.ts

```diff
@@ -15,7 +15,7 @@ export interface CoreOperators<T> {
   combineAll?: <R>(project?: (...values: Array<any>) => R) => Observable<R>;
   combineLatest?: <R>(...observables: Array<Observable<any> | ((...values: Array<any>) => R)>) => Observable<R>;
   concat?: <R>(...observables: (Observable<any> | Scheduler)[]) => Observable<R>;
-  concatAll?: () => Observable<any>;
+  concatAll?: () => Observable<T>;
```

Comment:
Oops, yes, you're correct - apologize. I've read type constraint incorrectly and thought `Observable.range<number>.map().concatall(): Observable<number | string>`, which is not since `map()` does passes type constraint correctly. 

Thanks for pointing out.